webapp/views/results-modal/script.js

```diff
@@ -127,11 +135,11 @@ class DeleteModal {
   onClickDelete() {
     deleteMethod('/output', { file: this.resultModal.resultToDelete }, () => new Promise((resolve) => {
       this.resultModal.remove(this.resultModal.resultToDelete);
```

Comment:
It's superfluous, the result will be gone after updateResults() has been called.